bin/docraptor

```diff
@@ -53,6 +53,14 @@ opt_parser = OptionParser.new do |opts|
     options[:async] = async
   end
 
+  opts.on("--callback_url url", "Set a callback URL to be notified upon success. Defaults to naught") do |callback_url|
```

Comment:
I think "defaults to naught" is confusing, would it be alright to just remove it?